src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -173,6 +173,9 @@
   <data name="ClearFiltersLink.Content" xml:space="preserve">
     <value>Clear filters</value>
   </data>
+  <data name="Clone.Text" xml:space="preserve">
+    <value>Clone</value>
```

Comment:
I think "duplicate" is more common for this use case.

Reply:
I thought about it and I ended up using Clone since the Duplicate term is already used to flag duplicate entries.
WDYT?

Reply:
Is this a problem? In this case we are referring to the action and not to the adjective. And after cloning an entry it is an duplicate and should be marked as duplicate.

Reply:
Make sense. I have updated the PR 😃